stdlib/array.gr

```diff
@@ -532,6 +582,9 @@ provide let reverse = array => {
  *
  * @param array: The array to convert
  * @returns The list containing all elements from the array
+ * 
+ * @example
+ * Array.toList([> 1, 2, 3, 4]) // [> 1, 2, 3, 4] becomes [1, 2, 3, 4]
```

Comment:
Same As Above:
```suggestion
 * Array.toList([> 1, 2, 3, 4]) == [1, 2, 3, 4]
```

Reply:
As we discussed yesterday....the inline is better for the noob or someone new to Grain.  You can clearly see the array format [<...] side-by-side to the list format [...].  Also by just putting the == without an inline ' // true' you have not stated what the answer is.